src/js/view/controls/controlbar.js

```diff
@@ -199,10 +197,7 @@ define([
                 elements.cc,
                 elements.audiotracks,
                 elements.playbackrates,
-                elements.mute,
                 elements.cast,
-                elements.volume,
-                elements.volumetooltip,
```

Comment:
@bartos So why is `volume` no longer here? Does `mute` now represent both of them?